gov/README.md

```diff
@@ -0,0 +1,151 @@
+# Governance Module
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Commands](#commands)
+  - [Query Commands](#query-commands)
+  - [Tx Commands](#tx-commands)
+  - [Run Using CLI](#run-using-cli)
+  - [Run Using REST](#run-using-rest)
+
+## Overview
+
+Heimdall governance works exactly the same as Cosmos-sdk's gov module. In this system, holders of the native staking token of the chain can vote on proposals on a 1 token = 1 vote basis. Here is a list of features the module currently supports:
```

Comment:
Maybe we should add any relevant links to cosmos sdk's gov module doc.

Reply:
I thought about that but then I looked at the latest gov module at https://docs.cosmos.network/main/build/modules/gov
And our's is using a bit older version, So dropped the idea.